src/index.html

```diff
@@ -7,6 +7,300 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img 
+          src="images\logo.png" 
+          alt="moyo_logo"
+          class="logo_icon"
```

Comment:
```suggestion
          class="logo__icon"
```
follow to BEM naming. it is element of logo block